src/server.cpp

```diff
@@ -956,8 +968,16 @@ JitterMeas.Measure();
     // one client is connected.
     if ( iNumClients > 0 )
     {
+        // ultra-low frequency updates
+        if ( iULFFrameCount > ULTRA_LOW_FREQUENCY_LIMIT )
+        {
+            iULFFrameCount = 0;
+
+            // things can go missing, so send this every now and then
+            CreateAndSendRecorderStateForAllConChannels();
+        }
```

Comment:
This is what I meant by the ultra-low frequency updates.

Reply:
I see. Have not recognized that. That ULTRA_LOW_FREQUENCY_LIMIT and also the LOW_FREQUENCY_UPDATE_INTERVAL has to be removed. We need normal messages. The status indicator is only for connected clients, not intended for the server list. So only the status change is transmitted, no continuous updates.

Reply:
You could take a look at PROTMESSID_MUTE_STATE_CHANGED. This is something similar. The Mute Icon status is only transmitted on a change.

Reply:
👍 OK.